libs/examples/src/indicator/indicator-default-example/indicator-default-example.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'dt-example-info-group-default',
+  templateUrl: 'indicator-default-example.html',
+})
+export class DtExampleIndicatorDefault {
+  indicatorState: boolean = true;
+
+  percentage: number = 90;
+
+  color: 'warning' | 'error' | null = null;
+
+  increaseValue(): void {
+    this.percentage += 10;
+    this.checkPercentage();
+  }
+
+  decreaseValue(): void {
+    this.percentage -= 10;
+    this.checkPercentage();
+  }
+
+  checkPercentage(): 'warning' | 'error' | null {
+    if (this.percentage >= 90 && this.percentage < 100) {
+      return 'warning';
+    } else if (this.percentage >= 100) {
+      return 'error';
+    } else {
+      return null;
+    }
+  }
+}
```

Comment:
I think we should make the example as simple as possible
```suggestion
  increaseValue(): void {
    this.percentage += 10;
    this.checkPercentage();
  }

  decreaseValue(): void {
    this.percentage -= 10;
    this.checkPercentage();
  }

  checkPercentage(): 'warning' | 'error' | null {
    if (this.percentage >= 90 && this.percentage < 100) {
      return 'warning';
    } else if (this.percentage >= 100) {
      return 'error';
    } else {
      return null;
    }
  }
}
```